test/snippet/CMakeLists.txt

```diff
@@ -23,7 +23,11 @@ macro (seqan3_snippet test_name_prefix snippet snippet_base_path)
         RUNTIME_OUTPUT_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/${snippet_target_path}"
     )
 
-    add_test (NAME "${test_name_prefix}/${snippet_test_name}_snippet" COMMAND ${target})
+    set (snippet_test_target "${test_name_prefix}/${snippet_test_name}_snippet")
+    add_test (NAME "${snippet_test_target}" COMMAND ${target})
+
+    # disable version checker, as it interferes with comparing the snippet output
+    set_tests_properties ("${snippet_test_target}" PROPERTIES ENVIRONMENT SEQAN3_NO_VERSION_CHECK=0)
```

Comment:
So this turns of the version check for all snippets correct? What about the argument parser snippets?

Reply:
> So this turns of the version check for all snippets correct?

Yes

> What about the argument parser snippets?

They of course, too.

If you wanted to ask whether we have snippets that "show" the behaviour of the version checker, that isn't the case as far as I can see.

Reply:
> If you wanted to ask whether we have snippets that "show" the behaviour of the version checker, that isn't the case as far as I can see.

Yes. And alright then